qa/integration/fixtures/certificates/certificate.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Good point, I think I wanted to make sure we have the same certificate so we can produce with it ?